HM3_task1.py

```diff
@@ -0,0 +1,7 @@
+numbers = [1, 2, 3, 4, 5, 6, 7, 8]
+for index, item in enumerate(numbers):
+    if item == 1 or item == 3 or item == 5 or item == 7:
```

Comment:
No-no-no!!!

Your code should work with any data. In this solution you just hardcoded values